### ISSUE TITLE
How to use Tap in Android?

### ISSUE THREAD

**Issue by gibo77** [callstack/react-native-slider#546]
## Ask your Question

Tapping the slider to change the value works with iOS, but not in Android. 
This is in the documentation called tapToSeek.
How can I make that work in Android by tapping?

'<View style={styles.OverAllView1}>'
       ' <Text style={styles.BrightnessText}>'
          Brightness: {SliderBrightness} %
        </Text>
        <View style={styles.SliderBrightness}>
          <View style={styles.ViewOff}>
            <Text style={styles.TextOff}>{'\n'}OFF</Text>
          </View>
          <View style={styles.OverallSlider}>
            <Slider
              style={styles.SliderMargin}
              step={1}
              disabled={!isEnabled} //If Glowing is activated, the Slider could not be changed.
            //  tapToSeek={true}
              minimumTrackTintColor="#307ecc"
              maximumTrackTintColor="#000000"
              maximumValue={100}
              minimumValue={0}
              value={SliderBrightness} //Updates on real time. Excellent.
              onSlidingComplete={val => {
                // console.log('OnSlidingComplete: ',SliderBrightness);
                ChangeBrightnessHappened = true;
                //setSliderBrightness(val);
                Gibo_Fetch('IGBT=' + val + '&Angga=' + ESPSerialNum);
              }}
              onValueChange={val => setSliderBrightness(val)}
            />
          </View>
          <View style={styles.OverAllTextON}>
            <Text style={styles.TextOn}>{'\n'}ON</Text>
          </View>
        </View>
        <View style={styles.OverAllBrightnessDescription}>
          <Text style={styles.BrightnessDescriptionText}>
            Slider at 0 value will turn OFF the light. 100 will turn ON. Tap the
            track to move the slider. If Dimming is active, Brightness can't be
            moved.
          </Text>
        </View>
      </View>

**Comment by BartoszKlonowski** [comment]
Hello @gibo77, thanks for reaching out!
The `tapToSeek` prop is only for iOS, because for Android, Windows and Web the tapping works out-of-the-box, by default, without any extra props or configuration.
Can you post any GIF of video presenting the issue with tapping on Android that you observe?

**Comment by gibo77** [comment]
Hi Team,
I was able to get a solution on my issue. Since I couldn't make the `useState` work inside the "OnFinishSliding", I just created my own function or callback and pass the `val`, as such `FuncSlider(val)`.
In the callback function I was able to update my variable `SliderBrightness` by equaling it to `val`, so that when I return back to home page I have the correct val data. 

This `val` parameter is not in the documentation. I just found out about it somewhere in an example by chance. It would be nice to have it in documentation.
